src/assets/templates/widgets/finaid_summary_netcost.html

```diff
@@ -4,41 +4,83 @@
+              Show less
+            </button>
+          </p>
         </th>
-        <td
-          class="cc-right"
-          data-cc-amount-directive="financialAidSummary.selected.totalGiftAidAndWaivers"
-          data-cc-amount-directive-color="false"
-        ></td>
+        <td class="cc-flex cc-flex-justify-end" data-cc-amount-directive="financialAidSummary.selected.totalGiftAidAndWaivers"></td>
       </tr>
-    </tbody>
-    <tfoot>
-      <tr>
-        <th>Net Cost</th>
+      <tr data-ng-if="financialAidSummary.selected.totalThirdPartyContracts">
+        <th>
+          <span>Third Party Contracts </span>
+          <button
+            data-ng-if="financialAidSummary.messages.thirdPartyMoreInfoMessage"
+            data-ng-click="financialAidSummary.showTipThirdParty();"
+            data-ng-show="!financialAidSummary.selected.tipVisibleThirdParty"
+            class="cc-button-link cc-text-line-height"
+          >
+            <i class="cc-icon fa fa-info-circle cc-icon-air-force-blue"></i>
+          </button>
+          <p data-ng-show="financialAidSummary.selected.tipVisibleThirdParty">
+            <span data-ng-bind-html="financialAidSummary.messages.thirdPartyMoreInfoMessage"></span>
+            <button data-ng-click="financialAidSummary.showTipThirdParty();" class="cc-button-link cc-text-line-height">
+              Show less
+            </button>
+          </p>
+        </th>
         <td
-          class="cc-right"
-          data-cc-amount-directive="financialAidSummary.selected.totalNetCost"
-          data-cc-amount-directive-color="false"
+          class="cc-flex cc-flex-justify-end"
+          data-cc-amount-directive="financialAidSummary.selected.totalThirdPartyContracts"
         ></td>
       </tr>
-    </tfoot>
+      <tr data-ng-if="financialAidSummary.selected.totalGiftAidAndWaivers">
+        <th>
+          <span class="cc-strong">Net Cost </span>
+          <button
+            data-ng-if="financialAidSummary.messages.netCostMoreInfoMessage"
+            data-ng-click="financialAidSummary.showTipNetCost();"
+            data-ng-show="!financialAidSummary.selected.tipVisibleNetCost"
+            class="cc-button-link cc-text-line-height"
+          >
+            <i class="cc-icon fa fa-info-circle cc-icon-air-force-blue"></i>
+          </button>
+          <p data-ng-show="financialAidSummary.selected.tipVisibleNetCost">
+            <span data-ng-bind-html="financialAidSummary.messages.netCostMoreInfoMessage"></span>
+            <button data-ng-click="financialAidSummary.showTipNetCost();" class="cc-button-link cc-text-line-height">
+              Show less
+            </button>
+          </p>
+        </th>
+        <td class="cc-strong cc-flex cc-flex-justify-end" data-cc-amount-directive="financialAidSummary.selected.totalNetCost"></td>
+      </tr>
+    </tbody>
   </table>
-</div>
+</div>
```

Comment:
Github wants an empty line at the end of the file. I try to appease the Github Gods.

Reply:
🗿All hail Github Gods 🗿